lib/rets/client.rb

```diff
@@ -266,6 +266,9 @@ def retrieve_metadata_type(type)
                         "ID"     => "0"
                       })
       res.body
+    rescue Rets::InvalidRequest => e
+      puts e.to_s
+      return ""  
```

Comment:
Could you create a new exception class to rescue just this error? I'm worried of all the other errors we would end up hiding. Some mlses only allow one connection and this error is raised if you try and establish a second connection. I'm pretty sure there are other legitimate errors that we would be hiding too, that's just the first one that comes to mind.